app/src/main/res/values/strings.xml

```diff
@@ -302,102 +302,103 @@
+    <string name="reply_from_another_account">Reply from different account</string>
+    <string name="launcher_icon_by">Thanks to フタバ for the app icon.</string>
+    <string name="ssl_bug_7_0">Android 7.0 can only use the elliptic curve \"prime256v1\", which your instance doesn\t support.\nUpgrade to Android 7.1.1 or later, or add support for elliptic curve \"prime256v1\" on your instance.\nSee https://code.google.com/p/android/issues/detail?id=224438</string>
     <string name="timeline_font">Timeline font (app restart required)</string>
-    <string name="timeline_font_bold">Timeline font (bold, app restart required)</string>
+    <string name="timeline_font_bold">Bold timeline font (app restart required)</string>
     <string name="restore_draft">Restore from draft</string>
     <string name="select_draft">Which draft to restore ?</string>
-    <string name="account_in_draft_is_lost">The account used for this draft has been removed from app. The following context can not be restored. in_reply_to, media attachment.</string>
-    <string name="reply_to_in_draft_is_lost">The reply origin referred by draft has been lost. The following context is removed. in_reply_to</string>
-    <string name="attachment_in_draft_is_lost">Some of media attachment in the draft have been lost.</string>
+    <string name="account_in_draft_is_lost">The account used to make this draft has been removed from the app. The following context can not be restored: in_reply_to, media attachment.</string>
+    <string name="reply_to_in_draft_is_lost">The reply origin referred by the draft has been lost. The following context is removed: in_reply_to</string>
+    <string name="attachment_in_draft_is_lost">Some of the media attachments in the draft have been lost.</string>
     <string name="app_data_export">Export app data</string>
     <string name="app_data_import">Import app data</string>
-    <string name="app_data_import_desc">You can export data to any app that can receive data, But maybe you can import data only from device\'s storage or Google drive.\nColumn background image may be not exported/imported over device.\nTimeline font is not exported/imported.\nCustom notification sound may be not exported/imported over device.</string>
-    <string name="app_data_import_confirm">existing data ( before import ) will be cleared. Are you sure?</string>
-    <string name="user_id_conversion_failed">User id conversion failed.</string>
+    <string name="app_data_import_desc">You can export data to any app that can receive data, but maybe you can import data only from device\'s storage or a cloud service.\nColumn background image may be not exported/imported over device.\nTimeline font is not exported/imported.\nCustom notification sound may be not exported/imported over device.</string>
```

Comment:
No, we can't use any cloud service. I tried some cloud service and only GD can be used.

Reply:
So the support is somehow specific to Google Drive?